source/plugins/go/src/oms.go

```diff
@@ -410,6 +412,20 @@ func createLogger() *log.Logger {
 	})
 
 	logger.SetFlags(log.Ltime | log.Lshortfile | log.LstdFlags)
+
+	// flush fluent-bit-out-oms-runtime.log at a reasonable rate (by default it's flushed like once a minute?)
+	TimeTicker := time.NewTicker(5 * time.Second)
```

Comment:
This change is entirely unrelated to detecting log loss, I'm happy to get rid of it. By default, some of the fbit log files are only flushed every few minutes which can make debugging a lot slower (since you have to wait for the log file to be flushed). This speeds that up